scripts/errcheck_excludes.txt

```diff
@@ -0,0 +1,8 @@
+// Don't flag lines such as "io.Copy(ioutil.Discard, resp.Body)".
+io.Copy
+// The next two are used in HTTP handlers, any error is handled by the server itself.
+io.WriteString
```

Comment:
This might work now, but not in every implementation that might be introduced soon. I will remove Copy and WriteString but up to you (:

Reply:
I copied this from prometheus/prometheus. :shrug: